### ISSUE TITLE
Canvas is distorted on Firefox

### ISSUE THREAD

**Issue by Tawpin** [upstage-org/limelight#179]
Since changing to Flexbox, the canvas can only be drawn in the top left and is zoomed in a lot.
![image](https://cloud.githubusercontent.com/assets/21133487/26814501/2d8d466e-4ada-11e7-89ba-733efcd0770a.png)

**Comment by rohrlach** [comment]
Thanks @Tawpin  we're aware of this.

**Comment by vickis** [comment]
hey tried testing on chrome and firefox
Chrome = no drawing appears to show up ... (tried resizing brush and changing brush colour to no effect...
firefox - cannot get past edit stage to actually enter stage ... so can't test 
:)
standing by

**Comment by rohrlach** [comment]
@vickis it's not deployed yet - Deployment happens on the hour - so in about 2 minutes

**Comment by vickis** [comment]
oh ha ha too keen!

**Comment by helenvarley** [comment]
seems possible to draw in any area of the canvas, using firefox on linux. however, when i resized the browser window, the drawing vanished! this is not good! :(

**Comment by rohrlach** [comment]
Yes I'm aware - there's a *ahem* fix *ahem in place for some situations, but canvas actually does this by design. 

We're going against the grain at this point

**Comment by helenvarley** [comment]
oh - that doesn't sound very promising! :(

**Comment by rohrlach** [comment]
It's the downside of being cutting edge :)

**Comment by vickis** [comment]
can we be just edgy? and working :)

**Comment by rohrlach** [comment]
@vickis @helenvarley - PR #187 is not inherently related to the resize issue, but it does robustly solve it.

**Comment by helenvarley** [comment]
good! :)

**Comment by helenvarley** [comment]
i just tested (in firefox only) & both drawing & avatar stayed visible when i resized the stage :+1: 
however, the proportions are not retained. if you make the stage narrower, the drawing and avatar become squished. :-1:
